content/en/docs/concepts/certificate.md

```diff
@@ -62,6 +62,39 @@ which contains the following explanation:
 > assumption that the remote end must already possess it in order to
 > validate it in any case.
 
+{{% alert title="Use ca.crt with caution" color="warning" %}}
+
+**When configuring a client** to connect to a TLS server with a serving certificate that is signed by a private CA,
+you will need to provide the client with the CA certificate in order for it to verify the server.
+If the client is running in a Kubernetes Pod _DO NOT MOUNT THE SAME SECRET AS THE SERVER_,
+to access the `ca.crt` file, because:
+
+1. That Secret also contains the private key of the server, which should only be accessible to the server.
+   You should use RBAC to ensure that the Secret containing the serving certificate and private key are only accessible to Pods that need it.
+2. When configuring the client you should independently choose and fetch certificate of the CA that you want to trust.
+   Download the  `ca.crt` out of band,
+   and store it in a Secret that is independent of the Secret populated by cert-manager.
```

Comment:
suggestion: The paragraph before the list that this appears in ends:

> DO NOT MOUNT THE SAME SECRET AS THE SERVER, to access the `ca.crt` file, because:

so this list should contain _reasons_ not to do that, I think. As phrased here, I don't think this is really a reason not to use `ca.crt`; it's just describing the best practice. 

How about this, adding a second reason not to use `ca.crt` and moving what's currently in here 2 down below with some minor rewording:

```suggestion
2. Rotating CA certificates safely relies on being able to have both the old and new CA certificates trusted at the same time. By consuming the CA directly from the source, this isn't possible; you'll be _forced_ to have some down-time in order to to rotate certificates.

When configuring the client you should independently choose and fetch certificates that you want to trust. Download `ca.crt` out of band and store it in a Secret or ConfigMap, totally independent of the Secret containing the server's certificate.
```